### PR TITLE
Merge 14.1 code freeze into trunk

### PR DESCRIPTION
**Pull request by jkmassel**
Includes:
- Production FluxC Version
- Release Toolkit (+ other ruby gems) update
- Release Notes
- Version number changes

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,4 +1,8 @@
 *** PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
+14.2
+-----
+
+
 14.2
```
Comment (mokagio) [inline RELEASE-NOTES.txt]: Something went unexpectedly here. 🤔 

From what I can make of the diff, the 14.2 header was already in the changelog but our automation didn't account for it and re-adde it?

```suggestion
14.2
```

File: Gemfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ gem 'nokogiri'
 
 ### Fastlane Plugins
 
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 7.0'
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 8.0'
```
Comment (mokagio) [inline Gemfile]: No big deal. But have you considered using `~> 8.1`, given that's the version we start using?

There's clearly no need for 8.1 exactly, as shown by the lack of changes in the fastlane automation in this diff, so I guess any `Gemfile` and `Gemfile.lock` combo that addresses the upgrade prompt will do.

**Review by mokagio — COMMENTED** [review]
- [x] New version header in `RELEASE-NOTE.txt` 
- [ ] Version updates in `build.gradle` – See note below
- [x] `release_notes.txt` updated with notes from `RELEASE-NOTE.txt` for current version 
- [x] `strings.xml` updates via automation merging values from libraries
- [x] Switched `ext` versions to stable releases

_Version updates in `build.gradle`_

Looks like we might need to ship a new FluxC version?

https://github.com/woocommerce/woocommerce-android/blob/eed205eb4966f3022a843edd97b1e10c3ed3b9bc/build.gradle#L98

**Comment by jkmassel** [comment]
Ah thanks for the FluxC catch! I cut the new FluxC version but forgot to update the code 🤦.

File: Gemfile (hunk 2 above)
Comment (jkmassel) [reply Gemfile]: Sure, doesn't hurt!

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (jkmassel) [reply RELEASE-NOTES.txt]: Should be fixed

**Review by mokagio — APPROVED** [review]
The diff from here doesn't make it clear, because we're on a `merge/` branch but just wanted to confirm the FluxC bump went into the `release/14.1` branch.

image

Thanks @jkmassel

**Pull request merged**